### ISSUE TITLE
Apple Login?

### ISSUE THREAD

**Issue by phys1ks** [oneall/social-login-vbulletin#6]
Will there ever be support for Social login with Apple on vBulletin? I paid for the Apple developer account and I can't add it to my vBulletin now?

**Comment by SchlesserClaude** [comment]
Hello,

Apple is already part of our services, we only need to make a small change to add add it to the Vbulletin plugin. This takes only a few minutes. Have you already installed our plugin?

Regards,

**Comment by phys1ks** [comment]
> Hello,
> 
> Apple is already part of our services, we only need to make a small change to add add it to the Vbulletin plugin. This takes only a few minutes. Have you already installed our plugin?
> 
> Regards,

Yes, I have the plugin installed and other services enabled, however Apple is not an option in the plugin.

![image](https://user-images.githubusercontent.com/64342930/82330910-07f15280-99a9-11ea-8024-26234841691c.png)

**Comment by phys1ks** [comment]
Hey there, any update on this?

**Comment by koodiabi** [comment]
Hello, apple login is not appear on vbulletin?

**Comment by xProjects-Development** [comment]
Please update...
